app/model_fetcher.py

```diff
@@ -1,20 +1,30 @@
+import json
 import logging
 import ntpath
 import os
 
+from flask import current_app as app
 from google.cloud import storage
 
 from app import model_config as mc
 
 
 class Fetcher(object):
-    """ Fetches classifier model files from Google Cloud Storage """
+    """ Fetches classifier model files from Google Cloud Storage."""
 
-    def __init__(self, source_config=mc.IN, dest_config=mc.OUT):
+    def __init__(self,
+                 config_path=os.path.join("static", "model_config.json"),
```

Comment:
nit: since you hardcode "./static/model_config.json" elsewhere, this seems like an excessive use of path.join ;)

Reply:
fair